crates/rune/src/ast/item.rs

```diff
@@ -10,6 +10,9 @@ pub enum Item {
     /// A use declaration.
     ItemUse(ast::ItemUse),
     /// A function declaration.
+    // large size difference between variants
+    // we should box this variant.
+    // https://rust-lang.github.io/rust-clippy/master/index.html#large_enum_variant
```

Comment:
Feel free to box the variant!

Reply:
That would require a lot of changes in a lot of places .. so I guess we could let that for another cleanup.